airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/streams/base_insight_streams.py

```diff
@@ -314,6 +314,10 @@ def _get_start_date(self) -> Mapping[str, pendulum.Date]:
                 start_date = min(start_date, refresh_date)
             else:
                 start_date = self._start_date
+
+            if start_date < self._start_date:
+                logger.warning(f"Ignore provided state and start sync from start_date ({self._start_date}).")
```

Comment:
nit: can we have the message like this?
```python
logger.warning(f"Starting the sync from: ({self._start_date}). The provided state: {%} was ignored.")
```

Not sure about the `state` mention in this context, but it seems like one of [start_date, self._start_date, oldest_date] ?